notebooks/models/logistic_regression_combined.py

```diff
@@ -264,6 +264,25 @@
     test_dfs[i]['Wind Angle'] = test_dfs[i]['Wind Angle'].interpolate(limit_direction='both')
     test_dfs[i]['Wind Speed'] = test_dfs[i]['Wind Speed'].interpolate(limit_direction='both')
 
+
+# Create Window Slice Features
+print('############### {} ###############'.format('Window Slice Features'))
+
+custom_params = {}
+week_columns = []
+for df in test_dfs + train_dfs:
+    for col in x_columns:
+        for i in range(4):
```

Comment:
Why the Magic number for the range, shouldn't the slicing of windows be dynamic?

Reply:
oh for sure. updated loop to be dynamic.